start-as-experiment.md

```diff
@@ -43,12 +43,10 @@ or are actively developing Open Source software.
 ## Solution
 
 Declare the InnerSource initiative as a time limited experiment. Define and
-communicate the criteria for projects to join the InnerSource experiment. The
-criteria should be chosen such that they maximize the chances of building a
-healthy InnerSource community around the selected InnerSource projects. They
-should also help to ensure that the setting of the projects is such that they
-can later be used to gain externally valid insights into the effects of
```

Comment:
I cannot really tell what was meant by "externally valid insights". So not sure if your changes reflect that correctly or not.

Reply:
To me the original text reads like it might have referred to [internal vs. external validity](https://www.verywellmind.com/internal-and-external-validity-4584479), i.e. generalizability.
In that regard I agree to @spier's comment.

A possible outcome where this might apply could be the desire to apply the insights learned in the experiment to other further improved experiments or later more "production" (bad wording, I know) InnerSource uses. In that regard one would want a valid experimental setup for the initial approach and interpreting the wording as illustrated above would make sense. 
What do you think @jpatrao?

Reply:
@lenucksi  Given your clarifying comments, I propose re-wording 
"The criteria, on examination within a project's context should transparently highlight the benefits of applying InnerSource."

as: 

"A set of criteria is good if the insights generated from it can safely be applied to other potential InnerSource projects."

Reply:
@jpatrao this sounds good to me. Go for it 👍

Reply:
@spier @lenucksi Done. Please have a look.